assets/app/views/_project-page.html

```diff
@@ -12,7 +12,15 @@
       </div>
     </div>
   </div>
-  <div class="sidebar-right">
-    sidebar-right
+  <div class="sidebar-right sidebar-pf sidebar-pf-right">
+    <div class="container-fluid">
+      <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
+        <div class="actions pull-right">
+          <a href="#"></a>
```

Comment:
I think steve added it as a reference of where sidebar-right header action links are supposed to go according to patternfly, i will comment out the line and include a comment